Feat(cv_deploy): Deploy manifest for CV Static Config Studio containers and configlets (#5824)
aristanetworks/ansible-avd: ansible_collections/arista/avd/roles/cv_deploy/README.md

    Make sure to enable "Studios - End-to-End Provisioning" under Settings, Features.

    ![Figure 1: Ansible Role arista.avd.cv_deploy](../../../../../docs/_media/studios_end_to_end_provisioning.png)

> - Currently only the first of the given cv_servers is being used.

Reviewer: why are you removing this?

Author: It was not rendered properly in the doc and `cv_servers` doesn't apply to cv_deploy. We use `cv_server` in cv_deploy.
<img width="1829" height="319" alt="image" src="https://github.com/user-attachments/assets/0a58ee46-7ff9-4784-819b-64fdfaa24185" />

cv_strict_system_mac_address: true
```

#### Static Configuration Studio deployment (Preview)

!!! warning "Preview Feature"

>     This feature is in preview. Everything is subject to change and **not officially supported for production environments**.

Reviewer: ```suggestion
    This feature is in preview. Everything is subject to change, not supported and may not be complete.
```

In addition to deploying device-specific configurations, the role allows for the deployment of a full hierarchy of containers and configlets to the CloudVision "Static Configuration" Studio. This is controlled by the following variables:

```yaml
# A list of dictionaries defining configlets to be created in the Configlet Library.
# Configlet names must be unique across all defined configlets.
cv_studio_configlets:
  - name: <str>
    file: <str>

# A list of dictionaries defining the root containers in the Static Configuration hierarchy.
# Container names must be unique among sibling containers (at the same level).
cv_studio_containers:
  - name: <str>
    description: <str, optional>
    tag_query: <str>
    match_policy: <str, default="match_all", choices=["match_all", "match_first"]>
    configlets:
      - name: <str>
    sub_containers:
      - name: <str>
        description: <str, optional>
        tag_query: <str>
        match_policy: <str, default="match_all", choices=["match_all", "match_first"]>
        configlets:
          - name: <str>
        sub_containers: <list of containers>
```

!!! note "Root Containers Order"

>     When initially deploying or adding new root containers, the role places its managed root containers to the top of the Studio container tree. Please be aware that this automated ordering **may displace any containers you have manually arranged**. This effect is purely visual and does not change how configurations are applied to devices.

Reviewer: It *will* affect the application order, which is top-down.
```suggestion
    When initially deploying or adding new root containers, the role places its managed root containers to the top of the Studio container tree. Please be aware that this automated ordering **may displace any containers you have manually arranged**.
```
